x/wasm/types/proposal.go

```diff
@@ -764,6 +818,18 @@ func validateProposalCommons(title, description string) error {
 	return nil
 }
 
+func NewUpdateInstantiateConfigProposal(
+	title string,
+	description string,
+	accessConfigUpdates []AccessConfigUpdate,
```

Comment:
personal preference:
```suggestion
	accessConfigUpdates ...AccessConfigUpdate,
```